spec/views/events/show.html.erb_spec.rb

```diff
@@ -211,6 +211,24 @@
     expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
   end
 
+  it "does not display send acceptances button after acceptances have been sent in selection phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
+    @event.acceptances_have_been_sent = true
```

Comment:
I think this flag could be set by factory creation, but I think it doesn't matter.